lib/dug/configurator.rb

```diff
@@ -9,6 +9,7 @@ class Configurator
     attr_accessor :token_store
     attr_accessor :application_credentials_file
     attr_accessor :unprocessed_label_name
+    attr_accessor :default_label
```

Comment:
I guess a more descriptive name would just be `processed_label_name`, as it's simply the opposite of the unprocessed one.